hack/verify_generated.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+if [[ -n "$(git status --porcelain .)" ]]; then
+        echo "uncommitted generated files. run 'make generate' and commit results."
```

Comment:
make generate ? this just update the boilerplate if I remember correctly

Reply:
This checks if there is something different from the underlying git versions.
We can use it together with generate in CI to check if the committed versions are the same we would generate.
For example here I use to ensure that the version in the text file is the same we have in manifests

Reply:
the check is good I was just commenting on the error message, running make generate won't do anything useful

Reply:
Ah yes, this comes from where I stole this :P 
Fixing it